lms/djangoapps/verify_student/views.py

```diff
@@ -459,7 +463,7 @@ def _redirect_if_necessary(
         if url is not None:
             return redirect(url)
 
-    def _get_verified_modes_for_course(self, course_key):
+    def _get_expired_verified_and_paid_mode(self, course_key):  # pylint: disable=invalid-name
         """Retrieve unexpired and expired verified modes for a course.
```

Comment:
This docstring is no longer consistent with the function it describes, and doesn't make the function's behavior clear.